content/en/docs/concepts/architecture/coordinated-leader-election.md

```diff
@@ -0,0 +1,56 @@
+---
+reviewers:
+- jpbetz
+title: Coordinated Leader Election
+content_type: concept
+weight: 200
+---
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="CoordinatedLeaderElection" >}}
+
+Kubernetes {{< skew currentVersion >}} includes an alpha feature that allows
+components to deterministically select a leader via Coordinated Leader Election.
+This is useful to satisfy Kubernetes version skew during cluster upgrades.
+Currently, the only supported selection strategy is `OldestEmulationVersion`,
+preferring the leader with the lowest emulation version, followed by binary
+version, followed by creation timestamp.
+
+## Enabling Coordinated Leader Election
+
+Ensure that `CoordinatedLeaderElection` [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled
+when you start the {{< glossary_tooltip text="API Server"
+term_id="kube-apiserver" >}}: and that the `coordination.k8s.io/v1alpha1` API is
+enabled.
+
+This can be done by setting `FEATURE_GATES="CoordinatedLeaderElection=true"` and
+`RUNTIME_CONFIG="coordination.k8s.io/v1alpha1=true"`.
```

Comment:
in which context do we support env vars for that? The general interface are command line flags.

Reply:
Updated to use flags instead of env vars.